index.bs

```diff
@@ -879,19 +921,23 @@ XRRay {#xrray-interface}
 An {{XRRay}} describes a geometric ray.
 
 <pre class="idl">
-[SecureContext, Exposed=Window]
+[SecureContext, Exposed=Window,
+ Constructor(optional DOMPointInit origin, optional DOMPointInit direction),
+ Constructor(XRRigidTransform transform)]
 interface XRRay {
   readonly attribute DOMPointReadOnly origin;
   readonly attribute DOMPointReadOnly direction;
-  readonly attribute Float32Array transformMatrix;
+  readonly attribute Float32Array matrix;
 };
 </pre>
 
 The <dfn attribute for="XRRay">origin</dfn> attribute defines the 3-dimensional point in space that the ray originates from, given in meters. The {{XRRay/origin}}'s {{DOMPointReadOnly/w}} attribute MUST be <code>1.0</code>. The <dfn attribute for="XRRay">direction</dfn> attribute defines the ray's 3-dimensional directional vector. The {{direction}}'s {{DOMPointReadOnly/w}} attribute MUST be <code>0.0</code> and the vector MUST be normalized to have a length of <code>1.0</code>.
 
-The <dfn attribute for="XRRay">transformMatrix</dfn> is a [=matrix=] which represents the transform from a ray originating at <code>[0, 0, 0]</code> and extending down the negative Z axis to the ray described by the {{XRRay}}'s {{XRRay/origin}} and {{direction}}.
+The <dfn attribute for="XRRay">matrix</dfn> attribute is a [=matrix=] which represents the transform from a ray originating at <code>[0, 0, 0]</code> and extending down the negative Z axis to the ray described by the {{XRRay}}'s {{XRRay/origin}} and {{direction}}.
+
+NOTE: The {{XRRay}}'s {{XRRay/matrix}} can be used to easily position graphical representations of the ray when rendering.
 
-NOTE: The {{XRRay}}'s {{XRRay/transformMatrix}} can be used to easily position graphical representations of the ray when rendering.
+ISSUE: Define {{XRRay}} constructor behaviors
```

Comment:
Why postpone this?  Is it complicated or contentious?

Reply:
Ah, no, I intended this as effectively a "Note to self" that I was going to address before committing. Which, as strategies go, apparently leaves something to be desired. 😜